custom_components/google_home/config_flow.py

```diff
@@ -49,10 +49,10 @@ async def async_step_user(self, user_input=None):
             username = user_input[CONF_USERNAME]
             session = async_create_clientsession(self.hass)
             client = GlocaltokensApiClient(
-                hass=self.hass,
+                self.hass,
+                session,
                 username=user_input[CONF_USERNAME],
                 password=user_input[CONF_PASSWORD],
```

Comment:
I think we should keep all arguments as `kwargs` keeping `hass=...` and `session=...`, I know it's not mandatory, but it's a good practice and is easier to follow the code :)

Reply:
I think `mypy` guarantees order of arguments is good here so I think this is optional.

Btw, it's may be interesting. Python 3.8 introduced positional only arguments in PEP 570: https://www.python.org/dev/peps/pep-0570/